<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AcquisitionChannelSettingsTab</class>
 <widget class="QWidget" name="AcquisitionChannelSettingsTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>472</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Roboto</family>
    <pointsize>-1</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font-family: Roboto; font-size: 14px;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QCheckBox" name="channelActive">
       <property name="font">
        <font>
         <family>Roboto</family>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>Active ▶️</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="useTriggerSource">
       <property name="text">
        <string>Trigger source ⚡</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>AFE</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <layout class="QFormLayout" name="formLayout_2">
        <property name="horizontalSpacing">
         <number>3</number>
        </property>
        <property name="verticalSpacing">
         <number>4</number>
        </property>
        <item row="2" column="0">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>DC offset</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>Obtained input range:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="obtainedInputRange">
          <property name="text">
           <string>5001 mV</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="dcOffsetCode">
          <property name="minimum">
           <number>-32768</number>
          </property>
          <property name="maximum">
           <number>32767</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>DC offset (mV)</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="dcOffsetMv">
          <property name="suffix">
           <string> mV</string>
          </property>
          <property name="minimum">
           <double>-10000.000000000000000</double>
          </property>
          <property name="maximum">
           <double>10000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>10.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Input range</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="inputRange">
          <property name="currentIndex">
           <number>5</number>
          </property>
          <item>
           <property name="text">
            <string>100 mV</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>250 mV</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>500 mV</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1000 mV</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2000 mV</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5000 mV</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>10000 mV</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Approx. offset (mV)</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="approximateOffset">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Trigger</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="5" column="0">
       <layout class="QFormLayout" name="formLayout">
        <property name="horizontalSpacing">
         <number>3</number>
        </property>
        <property name="verticalSpacing">
         <number>4</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Edge</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="triggerEdge">
          <item>
           <property name="text">
            <string>RISING</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>FALLING</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>BOTH</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Offset from</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QRadioButton" name="triggerOffsetFromZero">
            <property name="text">
             <string>0</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="triggerOffsetFromBias">
            <property name="text">
             <string>DC bias</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Offset mV</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Offset code</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,1">
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <item>
           <widget class="QSpinBox" name="triggerLevelCode">
            <property name="minimum">
             <number>-65535</number>
            </property>
            <property name="maximum">
             <number>65535</number>
            </property>
            <property name="singleStep">
             <number>64</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="triggerLevelCodeAbsolute">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,1">
          <item>
           <widget class="QDoubleSpinBox" name="triggerLevelMv">
            <property name="suffix">
             <string> mV</string>
            </property>
            <property name="minimum">
             <double>-10000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>10.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="triggerLevelMvAbsolute">
            <property name="text">
             <string>3123</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="triggerReset">
          <property name="maximum">
           <number>32767</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
